app/controllers/v1/vocabularies_controller.rb

```diff
@@ -0,0 +1,68 @@
+module V1
+  class VocabulariesController < ApplicationController
+    SHOW_FIELDS = [:string_key, :label, :custom_fields].freeze
+
+    # GET /vocabularies
+    def index
+      vocabularies = Vocabulary.all
+      render json: { vocabularies: vocabularies.map { |v| v.as_json(only: SHOW_FIELDS) } }.to_json
+    end
+
+    # GET /vocabularies/:string_key
+    def show
+      if (vocabulary = Vocabulary.find_by(string_key: params[:string_key]))
+        render json: vocabulary.to_json(only: SHOW_FIELDS), status: 200
+      else
+        render json: { error: { message: 'Not Found' } }.to_json, status: 404
```

Comment:
Based on the new JSON-API-like error format we talked about (for this, and for other controller error responses):
```
{
  errors: [
    {
      title: 'Not Found'
    }
  ]
}
```
See: https://jsonapi.org/format/#error-objects